<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDossierDialog</class>
 <widget class="QDialog" name="AddDossierDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Formation suivies</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="formationBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="addFormation">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/new/icons/add.png</normaloff>:/new/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteFormation">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/new/icons/bin.png</normaloff>:/new/icons/bin.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ajouter un semestre d’étude</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QTableWidget" name="tableSemestre">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="4">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="addSemestre">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/new/icons/add.png</normaloff>:/new/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteSemestre">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/new/icons/bin.png</normaloff>:/new/icons/bin.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="loginEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddDossierDialog</receiver>
   <slot>createDossier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddDossierDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addFormation</sender>
   <signal>clicked()</signal>
   <receiver>AddDossierDialog</receiver>
   <slot>formationAdded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addSemestre</sender>
   <signal>clicked()</signal>
   <receiver>AddDossierDialog</receiver>
   <slot>semestreAdded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteFormation</sender>
   <signal>clicked()</signal>
   <receiver>AddDossierDialog</receiver>
   <slot>formationRemoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteSemestre</sender>
   <signal>clicked()</signal>
   <receiver>AddDossierDialog</receiver>
   <slot>semestreRemoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createDossier()</slot>
  <slot>formationAdded()</slot>
  <slot>semestreAdded()</slot>
  <slot>semestreRemoved()</slot>
  <slot>formationRemoved()</slot>
 </slots>
</ui>
